### ISSUE TITLE
Critical Security Vunerabilities on Nginx docker image(1.19.10)

### ISSUE THREAD

**Issue by rahulsadanandan** [nginxinc/docker-nginx#534]
On scanning Nginx docker image version(image tag): 1.19.10-alpine I have noticed the below Vulnerabilitity tagged with Critical Severity.

```
{
   "user_issue_id":"1382170090968199168",
   "comp_id":"build://artifactory-pro-installers-docker-release:7.19.0-m009-20210414012920",
   "comp_name":"artifactory-pro-installers-docker-release",
   "comp_version":"7.19.0-m009-20210414012920",
   "type":"security",
   "targets":null,
   "issue_id":"XRAY-171859",
   "paths":[
      "default/builds/artifactory-pro-installers-docker-release"
   ],
   "summary":"libjpeg-turbo rdppm.c get_word_rgb_row() Function 16-bit PPM / PGM File Handling Heap Buffer Overflow",
   "severity":"Critical",
   "updated":"2021-04-14T03:13:23Z",
   "permissions":{
      
   },
   "is_source_root":false,
   "source":"3.13:libjpeg-turbo",
   "source_version":"2.0.6-r0",
   "source_id":"alpine://3.13:libjpeg-turbo"
}
```
[
<img width="1690" alt="Screenshot 2021-04-14 at 12 18 16 PM" src="https://user-images.githubusercontent.com/11055916/114672378-b2d9d000-9d22-11eb-8259-709ca604e276.png">
](url)

Please be aware of this issue, a possible fix was merged some days back here
https://github.com/libjpeg-turbo/libjpeg-turbo/commit/f35fd27ec641c42d6b115bfa595e483ec58188d2

**Comment by thresheek** [comment]
Thanks @rahulsadanandan, we'll wait when this is fixed in Alpine Linux 3.13.

**Comment by thresheek** [comment]
That being said @rahulsadanandan this cant be exploited via nginx, since the only supported file formats are JPEG, GIF, PNG, and WebP.  Of course, if you install additional software on top of this image, that might use affected file formats, it would be vulnerable.

**Comment by rahulsadanandan** [comment]
@thresheek Great.Thanks for that info.

**Comment by thresheek** [comment]
Fixed in `nginx:1.21.0-alpine`
